app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,49 @@
+<nav>
+ <div class="container">
+   <% if user_signed_in? %>
+    <%= link_to "Fakebook", authenticated_root_path, id: "logo", class: "logo-authenticated" %>
+   <% else %>
+    <%= link_to "Fakebook", unauthenticated_root_path, id: "logo" %>
+   <% end %>
+
+   <ul class="nav navbar-nav navbar-right">
+     <% if user_signed_in? %>
+       <div class="navbar-right">
+         <div class="navbar-right-links">
+           <%= link_to('Find Friends', find_friends_path) %>
+           <%= link_to('<i class="material-icons">notifications</i>'.html_safe, notifications_path) %>
+           <%= link_to('<img class="avatar" src="http://via.placeholder.com/48x48" alt="avatar">'.html_safe, user_path(1)) %>
+           <div class="nav-item dropdown">
+            <i class="material-icons" id="dropdown-toggle">keyboard_arrow_down</i>
+            <div id="dropdown-content" class="dropdown-menu" aria-labelledby="navbarDropdown">
+              <a class="dropdown-item" href="#">Action</a>
+              <a class="dropdown-item" href="#">Another action</a>
+              <%= link_to('Logout', destroy_user_session_path, method: :delete, class: "dropdown-item") %>
+            </div>
+          </div>
+         </div>
+       </div>
+     <% else %>
+       <div class="navbar-right">
+         <%= form_for(resource, as: resource_name, html: { class: "form-inline" }, url: session_path(resource_name)) do |f| %>
```

Comment:
What is the reason of naming user a resource? Resource is very generic name....
Why don't you just use User.new directly?

Reply:
Fixed :-)